tasks-by-users/Rehan-Khan-Engineer/07-Toggle-switch-css/style.css

```diff
@@ -0,0 +1,52 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Apply these properties in universal selector @Rehan-Khan-Engineer

Reply:
Done @Abbas4tech